modules/auxiliary/admin/ldap/ad_cs_cert_template.rb

```diff
@@ -0,0 +1,324 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::LDAP
+  include Msf::Auxiliary::Report
+
+  IGNORED_ATTRIBUTES = [
+    'dn',
+    'distinguishedName',
+    'objectClass',
+    'cn',
+    'whenCreated',
+    'whenChanged',
+    'name',
+    'objectGUID',
+    'objectCategory',
+    'dSCorePropagationData',
+    'msPKI-Cert-Template-OID',
+    'uSNCreated',
+    'uSNChanged',
+    'displayName',
+    'instanceType',
+    'revision',
+    'msPKI-Template-Schema-Version',
+    'msPKI-Template-Minor-Revision',
+  ].freeze
+
+  # LDAP_SERVER_SD_FLAGS constant definition, taken from https://ldapwiki.com/wiki/LDAP_SERVER_SD_FLAGS_OID
+  LDAP_SERVER_SD_FLAGS_OID = '1.2.840.113556.1.4.801'.freeze
```

Comment:
Not a blocker; similar to the thread of "we need a centralized place for SIDs - so I created one", we're close to having a similar requirement for OIDs 😄